src/components/v5/frame/NavigationSidebar/NavigationSidebar.tsx

```diff
@@ -176,7 +177,22 @@ const NavigationSidebarContent: FC<NavigationSidebarProps> = ({
             <div>{additionalMobileContent}</div>
           )}
         </div>
-        {logo && !isTablet && <div className="mt-auto">{logo}</div>}
+        {!isTablet && (
+          <>
+            <div className="w-11 px-px mt-auto flex justify-start">
+              <FeedbackButton
+                onClick={() => {
+                  if (openItemIndex === 0) {
+                    setOpenItemIndex(undefined);
+                  } else {
+                    setOpenItemIndex(0);
+                  }
+                }}
+              />
+            </div>
+            {logo ?? logo}
```

Comment:
did you mean to use conditional rendering here?

Reply:
Good catch! I went a bit further and just added the static logo. I've noticed that in every instance where this sidebar is used (which is 2), we always passed the same logo so I don't think we need the extra prop, at least not for now.

The specs also show the logo for every variation of this sidebar.